satellite.py

```diff
@@ -56,6 +60,55 @@
             9: 0.99,
             10: 0.25,
             11: 0.91}
+sampler = neal.Neal()
+
+if len(sys.argv) > 1:
+
+    num_satellites = 39
+    num_constellations = 13
+    coverage = {0: 0.90,
+                1: 0.82,
+                2: 0.92,
+                3: 0.14,
+                4: 0.43,
+                5: 0.62,
+                6: 0.78,
+                7: 0.48,
+                8: 0.94,
+                9: 0.21,
+                10: 0.97,
+                11: 0.61,
+                12: 0.11,
+                13: 0.93,
+                14: 0.65,
+                15: 0.10,
+                16: 0.87,
+                17: 0.64,
+                18: 0.29,
+                19: 0.96,
+                20: 0.23,
+                21: 0.45,
+                22: 0.56,
+                23: 0.31,
+                24: 0.09,
+                25: 0.85,
+                26: 0.34,
+                27: 0.80,
+                28: 0.75,
+                29: 0.41,
+                30: 0.66,
+                31: 0.55,
+                32: 0.01,
+                33: 0.30,
+                34: 0.15,
+                35: 0.60,
+                36: 0.76,
+                37: 0.70,
+                38: 0.84}
+    sampler = LeapHybridSampler()
```

Comment:
The sampler name might go to input file, or it can be a CLI option, which seems useful for experimenting. Adding more samplers should be straightforward.

Reply:
Yes, it is a CLI option now, and it should be easy, as long as we add the import statement.